tests/AMSlib/lb.cpp

```diff
@@ -46,28 +46,30 @@ int main(int argc, char *argv[])
   std::normal_distribution<double> distribution(0.5, 0.3);
   srand(rId);
   double threshold;
-  for ( int i = 0; i <= rId; i++){
+  for (int i = 0; i <= rId; i++) {
     threshold = distribution(generator);
   }
 
-  int computeElements = (threshold * SIZE); // / sizeof(double);
+  int computeElements = (threshold * SIZE);  // / sizeof(double);
```

Comment:
dead code?

Reply:
Isn't this used in line 70?

Reply:
Anyway this is a deprecated file. Don't think we are testing the load balance.